unlock-app/src/components/creator/FatalError.tsx

```diff
@@ -21,19 +33,15 @@ export const DefaultError = ({ title, children, illustration, critical }) => (
   </Container>
 )
 
-DefaultError.propTypes = {
-  illustration: PropTypes.string,
-  title: PropTypes.string,
-  children: PropTypes.node,
-  critical: PropTypes.bool,
-}
-
-DefaultError.defaultProps = {
-  illustration: '/static/images/illustrations/error.svg',
-  title: 'Fatal Error',
-  children: defaultError,
-  critical: true,
-}
+export const FallbackError = () => (
```

Comment:
Easier to add this separate component as the fallback than to compute default values in DefaultError (that would require optional properties and would remove some type system helpfulness)